01-js-week-1/js-testing.md

```diff
@@ -28,10 +30,10 @@ Jest tests are run from the command line using `npm test` (more on this later).
 
 ### Setup
 
-We will install the Jest Command-line Interface globally.  This will allow us to run our tests from the terminal similar to how we tested in Ruby.  To install the Jest CLI execute the command:
+To add jest to a node project you can add it to your development dependencies (you don't need the test suite when you deploy the app) with:
```

Comment:
Will this mean that they need to execute this command from the project repo directory and that they will need to redo this in every project? 
If so, is there a reason why we're moving away from installing it globally? If so, can we name both of those notes about where and when they'll need to run this command?

Reply:
This is "jest" so that they know how to add a dependency to package.json.  For their projects it's already good to go.  

jest-cli is only needed if you want to use jest without a package.json, which none of our projects do.  So I took it out.